features/mbedtls/targets/TARGET_CRYPTOCELL310/mbedtls_device.h

```diff
@@ -23,4 +23,7 @@
 
 #define MBEDTLS_ENTROPY_C
 
+#define MBEDTLS_SHA1_ALT
+#define MBEDTLS_SHA256_ALT
```

Comment:
I understand, but basically, it's same implementation, except for the SHA mode, so I thought of doing them together